runtime/cudaq/qis/qubit_qis.h

```diff
@@ -791,6 +791,45 @@ std::vector<measure_result> mz(QubitRange &q) {
   return b;
 }
 
+// Measure all qubits in the range, return vector of 0,1
+#if CUDAQ_USE_STD20
+template <typename QubitRange>
+  requires std::ranges::range<QubitRange>
+#else
+template <
+    typename QubitRange,
+    typename = std::enable_if_t<!std::is_same_v<
+        std::remove_reference_t<std::remove_cv_t<QubitRange>>, cudaq::qubit>>>
+#endif
+std::vector<measure_result> my(QubitRange &q) {
+  std::vector<measure_result> b;
+  for (auto &qq : q) {
+    r1(-M_PI_2, qq);
```

Comment:
Here I think you don't want the basis change operations, since you are calling the single qubit `my` which will apply the basis change already.

Reply:
Will do; thanks for the tip!